CRM/Sumfields/Form/SumFields.php

```diff
@@ -97,6 +102,13 @@ function buildQuickForm() {
 
     $this->assign('fieldsets', $fieldsets);
 
+    $bd_label = ts('Performance', array('domain' => 'net.ourpowerbase.sumfields'));
+    $bd_options = array(
+      'default' => ts("Default settings", array('domain' => 'net.ourpowerbase.sumfields')),
+      'big_data' => ts("Disable triggers (increases performance on large installations)", array('domain' => 'net.ourpowerbase.sumfields'))
+    );
+    $this->addRadio('performance_settings', $bd_label, $bd_options);
+
```

Comment:
How about if we try to make the label and variable more consistent throughout? I would suggest the label be: "How often should summary data be updated?" The variable name would be: 'data_update_method' and The options would be:
'via_triggers' - ts("Instantly")
'via_cron' - ts("When ever the cron job is run (increases performance on large installation)"

Reply:
Sure, I agree, I'll prep it right away!